2021-04-26 07:43:11,858 INFO    MainThread:9020 [wandb_setup.py:_flush():69] setting env: {}
2021-04-26 07:43:11,859 INFO    MainThread:9020 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-26 07:43:11,860 INFO    MainThread:9020 [wandb_init.py:_log_setup():336] Logging user logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210426_074311-qgvtqqtz/logs/debug.log
2021-04-26 07:43:11,860 INFO    MainThread:9020 [wandb_init.py:_log_setup():337] Logging internal logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210426_074311-qgvtqqtz/logs/debug-internal.log
2021-04-26 07:43:11,860 INFO    MainThread:9020 [wandb_init.py:init():369] calling init triggers
2021-04-26 07:43:11,860 INFO    MainThread:9020 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-26 07:43:11,861 INFO    MainThread:9020 [wandb_init.py:init():418] starting backend
2021-04-26 07:43:11,861 INFO    MainThread:9020 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-26 07:43:11,925 INFO    MainThread:9020 [backend.py:ensure_launched():123] starting backend process...
2021-04-26 07:43:11,973 INFO    MainThread:9020 [backend.py:ensure_launched():127] started backend process with pid: 9106
2021-04-26 07:43:11,975 INFO    MainThread:9020 [wandb_init.py:init():423] backend started and connected
2021-04-26 07:43:11,978 INFO    MainThread:9020 [wandb_init.py:init():465] updated telemetry
2021-04-26 07:43:11,979 INFO    MainThread:9020 [wandb_init.py:init():484] communicating current version
2021-04-26 07:43:13,697 INFO    MainThread:9020 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-26 07:43:13,697 INFO    MainThread:9020 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-26 07:43:13,848 INFO    MainThread:9020 [wandb_init.py:init():522] starting run threads in backend
2021-04-26 07:43:17,008 INFO    MainThread:9020 [wandb_run.py:_console_start():1524] atexit reg
2021-04-26 07:43:17,011 INFO    MainThread:9020 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-26 07:43:17,012 INFO    MainThread:9020 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-26 07:43:17,015 INFO    MainThread:9020 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-26 07:43:17,016 INFO    MainThread:9020 [wandb_init.py:init():546] run started, returning control to user process
2021-04-26 07:44:40,843 INFO    MainThread:9020 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-26 07:44:40,850 INFO    MainThread:9020 [wandb_run.py:_restore():1466] restore
2021-04-26 07:44:43,013 INFO    MainThread:9020 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 547
  total_bytes: 547
}

2021-04-26 07:44:44,020 INFO    MainThread:9020 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 547
  total_bytes: 934
}

2021-04-26 07:44:44,189 INFO    MainThread:9020 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 547
  total_bytes: 6087
}

2021-04-26 07:44:44,292 INFO    MainThread:9020 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6087
  total_bytes: 6087
}

2021-04-26 07:44:44,395 INFO    MainThread:9020 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6087
  total_bytes: 6087
}

2021-04-26 07:44:44,499 INFO    MainThread:9020 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6087
  total_bytes: 6087
}

2021-04-26 07:44:44,602 INFO    MainThread:9020 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6087
  total_bytes: 6087
}

2021-04-26 07:44:44,704 INFO    MainThread:9020 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6087
  total_bytes: 6087
}

2021-04-26 07:44:46,166 INFO    MainThread:9020 [wandb_run.py:_show_files():1838] logging synced files
